cmd/agent/install_script.sh

```diff
@@ -343,6 +344,14 @@ else
   $sudo_cmd chmod 640 $CONF
 fi
 
+# Creating or overriding the install information
+$sudo_cmd echo "---
+install_method:
+  tool: install_script
+  tool_version: install_script
+  installer_version: install_script-$install_script_version
+" > $ETCDIR/install_info
```

Comment:
`sudo echo "..." > $ETCDIR/install_info` won't solve the `permission denied` problem on `$ETCDIR/install_info` as the `sudo` is applied to the `echo` command, and not to the file write.
You need to do something like this:
`echo "..." | $sudo_cmd tee $ETCDIR/install_info >/dev/null` (the `>/dev/null` is there to prevent `tee` from printing to stdout).

Reply:
➕

Reply:
Updated it as it's done in other part of the script